docs/howto/account-billing/forgot-your-password.md

```diff
@@ -0,0 +1,55 @@
+# Forgot your password or username
+
+Via [{{gui}}](https://{{gui_domain}}) you can reset lost password or reclaim your username.
+
+## Reset lost password
+
+Navigate to <https://{{gui_domain}}> and click _Forgot your password?_.
+
+![Picture with a orange button with white text Forgot your password? framed with red color](assets/Forgot-your-password.png)
+
+This image appears. Enter your username and click _Reclaim password!_.
```

Comment:
Suggestion: "A pop-up window titled _Reclaim lost password!_, appears. Type in your username, then click on the _Reclaim password!_ button."